apps/user_profile/views.py

```diff
@@ -42,3 +43,16 @@ class RegisterFormView(FormView):
     def form_valid(self, form):
         form.save()
         return super(RegisterFormView, self).form_valid(form)
+
+
+def child_form(request):
+    if request.method == 'POST':
+        form = RegisterChildForm(request.POST)
+        if form.is_valid():
+            form.save()
+            return redirect('/success/')
```

Comment:
do you have `success` url?

Reply:
it's not like url, after successful registration it shows pop-up that redirect to user page